### PR TITLE
Purge: ./vendor

### PR DESCRIPTION
**Pull request by yashvardhan-kukreja**
Because vendoring wasn't serving much of a substantially great purpose and was instead leading to dirty PRs and significantly larger codebase to deal with.

Signed-off-by: Yashvardhan Kukreja <ykukreja@redhat.com>

### CODE REVIEW

**Comment by yashvardhan-kukreja** [comment]
image

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-exclude: "vendor/ | *.deepcopy.go"
+exclude: " *.deepcopy.go"
```
Comment (erdii) [inline .pre-commit-config.yaml:1]:
```suggestion
exclude: "*.deepcopy.go"
```

Comment (yashvardhan-kukreja) [reply .pre-commit-config.yaml]: Whoops!

**Pull request merged**